frontend/src/App.tsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import logo from './logo.svg';
```

Comment:
I have reverted to prettier now but every time I save it will format the imports in the same order as here. I also tried setting up prettier as my default formatter but the outcome is the same. Have you maybe forgotten to format this file beforehand?

Reply:
Yes of course, like mentioned in the comment below. So nothing to see here